app/views/jobs/index.html.erb

```diff
@@ -80,6 +80,10 @@
           <span class="glyphicon glyphicon-trash"></span> Remove failed jobs
         <% end %>
 
+        <%= link_to retry_queued_jobs_path, class: "btn btn-default", method: :post do %>
+          <span class="glyphicon glyphicon-refresh"></span> Retry Queued Jobs
```

Comment:
Let's do `Retry queued jobs` to match case like the other buttons.  (Or update them to have capitals, if you'd prefer!)